### PR TITLE
Configuration variable to disable command inherit

### PR DESCRIPTION
**Pull request by dbeniamine**
Hi,

I also added a configuration variable to disable command inherit:

    URxvt.tabbed.new-tab-inherit-command: false

### CODE REVIEW

File: tabbedex [hunk 1]
```diff
@@ -343,10 +343,12 @@ sub new_tab {
 
    push @urxvt::TERM_EXT, urxvt::ext::tabbedex::tab::;
 
+   my $argv=$self->{argv};
+   $self->{argv}="" unless ($self->{new_tab_inherit_command});
```
Comment (mina86) [inline tabbedex]: I’m confused. How is it supposed to work? The second time this function is executed, $self->{argv} will be a string thus $argv will be a string thus @{ $argv } will fail. Is there something I’m missing, or perhaps what you actually meant was:
if (!$self->{new_tab_inherit_command}) { $argv = []; }
?

**Comment by dbeniamine** [comment]
You are right it should be `$self->{argv}=[]` instead of `$self->{argv}=""`. The idea is just to forgot the argv as soon as they are not required anymore.

**Comment by mina86** [comment]
That makes sense alas I think it would be more obvious what this is doing if self->{argv} was cleared  in on_start.

**Comment by dbeniamine** [comment]
Done, I also took care of merging the master branch :)

**Comment by mina86** [comment]
I took the liberty of renaming the variable to read ‘inherits’ instead of ‘inherit’.

**Comment by mina86** [comment]
FYI, I’ve just pushed a commit which changes this configuration from a boolean new-tab-inherits-command to more flexible new-tab-command option.

**Pull request closed without merging**